reports/2017-06-09.md

```diff
@@ -0,0 +1,35 @@
+# Development Report for June 09, 2017
+
+## Docker Schema 1 Pull
+
+We added support for pulling Docker images which use the schema1 manifest
+format in the containerd client. This allows pulling images from older
+registries as well as images pushed from older versions of Docker. With this
+change any image from the Docker Hub or registry supporting the v2 API can be
+pulled. As part of our commitment to support OCI images, the schema 1 images
+pulled by the client are converted to OCI images before being stored by
+containerd. The containerd client will only support pushing these images as
+OCI.
+
+[#968 Schema 1 pull](https://github.com/containerd/containerd/pull/968)
+
+## Namespaces for multi-tennancy
```

Comment:
nit spelling: "*multitenancy*" although I think this is a fake [IT-sphere word creation](https://en.wikipedia.org/wiki/Multitenancy) since most spelling checkers underline it :)